packages/pattern-picker/src/use-link-in-bio-designs.ts

```diff
@@ -0,0 +1,51 @@
+import { useStarterDesignsQuery } from '@automattic/data-stores';
+import { useLocale } from '@automattic/i18n-utils';
+import { useQuery } from 'react-query';
+import type { Design } from '@automattic/design-picker';
+
+/**
+ * This query isn't really a query, it's just meant to use Calypso's react-query async storage to store the order for the user forever
+ *
+ * @param length the length of the array to order
+ * @returns an array of numbers in a random order
+ */
+function useOrder( length: number ) {
```

Comment:
This is copied from https://github.com/Automattic/wp-calypso/blob/8e3568b66d40f4eefd604e35b903f8f7435d6878/packages/pattern-picker/src/use-query-patterns.ts#L12-L26